polaris-service/src/test/java/org/apache/polaris/service/admin/PolarisServiceImplIntegrationTest.java

```diff
@@ -379,6 +382,73 @@ public void testCreateCatalogWithInvalidName() {
     }
   }
 
+  @Test
+  public void testCreateCatalogWithAzureStorageConfig() {
```

Comment:
It might also be useful to add CLI tests

Reply:
CLI tests are fine, but I think the primary purpose of CLI tests is to evaluate that the CLI behaves as expected. For the purpose of this PR, we're fixing an API issue, so I think testing the API directly makes the most sense. If the CLI isn't passing something to the API, that's a concern, but I don't think we need to duplicate every API test in the CLI. Moreover, the CLI test doesn't have the infrastructure for setting fake google credentials, so adding tests there would require a pretty significant change and I don't know that we'd get much more value for the extra work.

Reply:
Adding CLI or Python client tests would add further validation that the types (i.e. `AzureStorageConfigInfo`) serialize and deserialize correctly in the Python client.

Reply:
As above, if we don't have tests for that, I don't think we ought to block a bugfix from being checked in to address that issue. I think de/serialization tests are great. Let's open an issue and tackle it in another PR.

Reply:
Being that the issue was initially reported as affecting the CLI, I still think it makes sense to add CLI tests.